fastlane/Fastfile

```diff
@@ -0,0 +1,106 @@
+fastlane_version "0.4.2" 
+
+lane :build_for_test do
+  xcodebuild(
+    scheme: 'Kiosk',
+    destination: "OS=8.1,name=iPad Air",
+  )
+end
+
+lane :test do
+  xctest(
+    destination: "OS=8.1,name=iPad Air",
+  )
+end
+
+lane :oss do
+  keys = ['ArtsyAPIClientSecret', 
+    'ArtsyAPIClientKey', 
+    'HockeyProductionSecret', 
+    'HockeyBetaSecret', 
+    'MixpanelProductionAPIClientKey', 
+    'MixpanelStagingAPIClientKey', 
+    'CardflightAPIClientKey', 
+    'CardflightAPIStagingClientKey', 
+    'CardflightMerchantAccountToken', 
+    'CardflightMerchantAccountStagingToken', 
+    'BalancedMarketplaceToken', 
+    'BalancedMarketplaceStagingToken']
+
+    commands = keys.map { |key|
+      command = "bundle exec pod keys set #{key} '-'"
+      if key == keys.first
+        command += " Eidolon"
+      end
+
+      command
+    }.join(' ; ')
+
+    sh "cd .. ; #{commands} ; pod install"
```

Comment:
We could use the built in [cocoapods action](https://github.com/KrauseFx/fastlane/blob/master/Actions.md#cocoapods) :+1: